app/views/reminder_mailings/_form.html.slim

```diff
@@ -0,0 +1,62 @@
+= simple_form_for @reminder_mailing do |f|
+  = simple_error_notice f
+  = f.input_field :kind, value: @reminder_mailing.kind, as: :hidden
+
+  = render 'form_submit_back', f: f
+
+  .row
+    .col-xs-12.col-md-4
+      .panel.panel-default
+        .panel-heading
+          strong Erklärung
+        .panel-body
+          'Sie können die folgenden Platzhalter benützen:
+
+        ul#template-variables.list-group
+          - ReminderMailing::TEMPLATE_VARNAMES.each do |var_name|
+            li.list-group-item %{#{var_name}}
+
+        .panel-body
+          p Zum Beispiel:
+          pre
+            = "Guten Tag %{Anrede} %{Name}\r\n"
+            = "Wir bitten um Ihre Rückmeldung für:\r\n%{Einsatz}.\r\n"
+            = "Danke im Voraus!\r\n"
+          p Resultat:
+          pre
+            = "Guten Tag Frau Marie Muster\r\n"
+            = "Wir bitten um Ihre Rückmeldung für:\r\nJogakurs - Zentrum Juch\r\n"
+            = "Danke im Voraus!\r\n"
+    .col-xs-12.col-md-8
+      fieldset
+        legend Betreff
```

Comment:
since `Subject`& `Body` are required, and you have the legend plus the labels(which are shown yet still in english), I would suggest to go for this:

```
fieldset
    legend * Betreff
    = f.input :subject, label: false
fieldset
    legend * Text
    = f.input :body, label: false
```

then you don't have twice the same labels, and still the User knows, that they're required